lib/recase.dart

```diff
@@ -21,15 +22,23 @@ class ReCase {
       String nextChar = (i + 1 == text.length
           ? null
           : new String.fromCharCode(text.codeUnitAt(i + 1)));
-
+      String nextNextChar = (i + 2 >= text.length
```

Comment:
I am not totally pleased with this variable name.  If you know a better word for "the thing the comes after the next thing" then I'd rather use it.